### ISSUE TITLE
issue with dashboard report while execute failed-testcases more then one time

### ISSUE THREAD

**Issue by narendrarajput** [qmetry/qaf#205]
### QAF Version
> qaf and support v2.1.14-RC1

### Scenario to reproduce
**Step 1:**
While I'm executing the test lets say there is only one BDD file and only one scenario which is going to fail
So first time if i see the execution in dashboard it is showing everything fine there. (count and scenario)

**Step 2:**
If navigate to `/target>surefire-reports` and execute the `testng-failed-qas.xml`  then after complete execution (test still failed) if view the report then it show the correct count and correct name of scenarios which failed this time as well.

**Step 3:** 
If again execute  `testng-failed-qas.xml`   and view the dashboard then it shows the correct count on overview but in detailed report it added the same failed testcases again(duplicate)
 
**Note :** If again run the same `testng-failed-qas.xml` then it show the same wrong details for scenario. Scenario observed with only one testcase if there is multiple testcases then its very hard to understand what's in the report

### Expected behavior
It show correct count and test scenario detail for each failed test execution summary

### Actual behavior

Adding the same test scenario again if execute fail test from `testng-failed-qas.xml` more then once

Refer the snap 

![image](https://user-images.githubusercontent.com/14977251/42573364-8ce67582-8539-11e8-8cdb-7fad8fca2d0f.png)

**Comment by cjayswal** [comment]
It looks same directory used for saving test results used. This happens when you are using maven and instead of running maven test, running as TestNG through IDE.

**Comment by narendrarajput** [comment]
Yes I'm running failed test as TestNGSuite. Is there a way to execute that `testng-failed-qas.xml` as Maven Test ? If I'm configuring new run configuration then its showing testng configuration only.

**Comment by cjayswal** [comment]
You should refer maven documentation for that. Try by passing system property while executing maven.
